### DISCUSSION TITLE
Add a numerical value of errors on the result page

### DISCUSSION THREAD

**Discussion by grigoriynikolaev** [monkeytypegame/monkeytype#4077] [Ideas] [answered]
Hello.
Is there a better way to know how many letters you mistype and then corrected other than manually count red dots on the graph, multiplying them dependently on the position on the x-axis? Maybe there is a setting I am missing?

I know, accuracy percentage is a better way to track your error rate especially if you often switch between various tests, but I usually do one preset(for example 1k-eng,30w,punctuation) for a couple days and then switch it when I get bored. So error count would be preferable.

On the results page page my typical 'characters' string looks like 143/0/0/0, and the only value that is changing is overall character count. If you correct your errors on a go(for "clean" tests), "incorrect", "missing" and "extras" values are useless, so way can't I have error count here? Or when you mouse over your acc percentage?

**Comment by Miodec** [comment, marked as answer]
Just to confirm - you're looking for a way to get the number of incorrect keypresses? Even if you corrected all your mistakes?

**Reply by grigoriynikolaev** [reply]
Yes, exactly. But judging by absence of similar discussions/issues I may be the only one who's bothered by that. So it seems like a pretty niche problem.

**Reply by Miodec** [reply]
I can add it to the popup when hovering over the accuracy number

**Reply by Miodec** [reply]
![image](https://user-images.githubusercontent.com/13181393/225888043-901dde41-ac68-4759-97fd-8fe9f216a661.png)

**Reply by grigoriynikolaev** [reply]
Thank you very much.
Now I can just rest mouse cursor in the accuracy percentage area and see a number of mistakes without reaching for the mouse each time.
